src/notyf/toast-container/toast-container.css

```diff
@@ -0,0 +1,8 @@
+:host { 
+        border: 1px solid;
```

Comment:
I think it's not necessary to set a border to the container, at least notyf doesn't do it. I might be missing something here, though, so I'll leave it up to you.